opp_gui/src/widgets/tool_path_parameters_editor_widget.cpp

```diff
@@ -190,11 +306,80 @@ void ToolPathParametersEditorWidget::onGenerateToolPathsComplete(
       tp.params.config.surface_walk_generator.min_segment_size = ui_->double_spin_box_min_segment_length->value();
       tp.params.config.surface_walk_generator.generate_extra_rasters =
           false;  // No option to set this from GUI at present.
-      tp.params.config.surface_walk_generator.raster_wrt_global_axes =
-          false;  // No option to set this from GUI at present.
+      false;      // No option to set this from GUI at present.
       tp.params.config.surface_walk_generator.intersection_plane_height =
           ui_->double_spin_box_intersecting_plane_height->value();
 
+      tp.params.config.plane_slicer_generator.point_spacing = ui_->double_spin_box_point_spacing->value();
+      tp.params.config.plane_slicer_generator.tool_offset = ui_->double_spin_box_tool_z_offset->value();
+      tp.params.config.plane_slicer_generator.raster_spacing = ui_->double_spin_box_line_spacing->value();
+      tp.params.config.plane_slicer_generator.raster_rot_offset =
+          ui_->double_spin_box_raster_angle->value() * M_PI / 180.0;
+      tp.params.config.plane_slicer_generator.min_hole_size = ui_->double_spin_box_min_hole_size->value();
+      tp.params.config.plane_slicer_generator.min_segment_size = ui_->double_spin_box_min_segment_length->value();
+
```

Comment:
same response. Not used, but may want to revert without having to do a lot of work. If the parameters were the same for both planners, there wouldn't be much work. Since they are different, it would take quite a bit of time to revert.